pages/myposts.vue

```diff
@@ -244,314 +255,250 @@
     </b-container>
   </client-only>
 </template>
-<script>
+
+<script setup>
 import { useRoute } from 'vue-router'
 import pluralize from 'pluralize'
 import dayjs from 'dayjs'
-import { useGroupStore } from '../stores/group'
```

Comment:
Group Store, Compose Store and some others seem not to be used really - **removed**.

Reply:
Agree.